lib/helpers/sassNodeModulesAliasImporter.js

```diff
@@ -0,0 +1,49 @@
+var path = require("path");
+var findNodeModules = require("find-node-modules");
+var fs = require("fs");
+
+var NODE_MODULES_ALIAS = "~";
+var sassFileExtensions = [".sass", ".scss"];
```

Comment:
We have these extensions somewhere else also (maybe not seperately). I think it would be better to have them in one place.